### ISSUE TITLE
Crash upon export

### ISSUE THREAD

**Issue by EivindSul** [guillaumechereau/goxel#339]
Hi, I'm using goxel on a wayland arch linux desktop.

Whenever I export an image, goxel crashes and disappears, and goes to 100% CPU usage. I closed it using htop.
If I try to reopen my file from within goxel, the same crash happens. It feels like the file has been broken somehow. 
If I reopen the file from the terminal by typing "goxel filename.gox", it works fine. It still crashes when exporting, leaving no error message in the terminal. Maybe this is in a log file?

So in other words, exporting images crashes the program and makes it so that I have to reopen the file from the terminal, not from "open" in the menu. The exported files are completely fine. I am able to work around it, but it is definitely worth looking into.

**Comment by guillaumechereau** [comment]
Hello, what version of goxel are you using?  Did you compile it yourself?

There are no log files, so if you see nothing in the terminal it means it crashed without any message.

**Comment by EivindSul** [comment]
I used the arch package. It says goxel 0.12.0. Should I compile it myself?

**Comment by guillaumechereau** [comment]
First, you can try the AppImage from the last commit:

https://github.com/guillaumechereau/goxel/actions/runs/7438929595

I am not sure it will work on Arch though (I use ubuntu).

If that still fails, yes, it would be nice if you could get the source from github and compile it yourself (in debug).  Then you might see more logs, or you can run it with gdb and see the stack trace.  If you need help let me know.

**Comment by EivindSul** [comment]
I'll try compiling it myself then. Thanks for the quick reply!

**Comment by EivindSul** [comment]
I get a wayland error message when I tried to build it myself. No idea if i may have done something wrong, I just cloned from git and used "make". I figured the dependencies would still be installed from my previous install, maybe I should change to X11 version of glfw.
![20240108_10h58m34s_grim](https://github.com/guillaumechereau/goxel/assets/62134330/31efe780-87dd-4c4d-9a70-6422b72de16e)
I got the appimage to work correctly though, it appears to be a newer version and does not have the problem with crashing.
